src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -162,12 +188,19 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <ProduceReferenceAssembly Condition="'$(ProduceReferenceAssembly)' == ''">false</ProduceReferenceAssembly>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(OutputPath)' == '' ">
+  <!--
+    For projects loaded with MPFProj (Legacy project system) that defines the properties per Configuration/Platform
+    combination by the IDE itself, the validation of an invalid Configuration/Platform is determined by the non-empty
+    value of the 'OutputPath' property specified under the IDE generated Configuration/Platform 'PropertyGroup' section.
```

Comment:
I don't understand this comment. What does the project system have to do with the validity computation--isn't that entirely in MSBuild logic?

Reply:
Nothing. It's just that the project uses this logic typically load in legacy project system and not on CPS.

Also see: https://github.com/dotnet/msbuild/pull/5238#discussion_r503230153

As I said in the above comment I'm open to a better alternative term.

Reply:
How do you feel about the wording in c1a2ff8c68?

Reply:
Not bad but I do prefer the previous but with changes like...

`For projects loaded by Legacy project system with only common targets...`

Either we could use this sentence directly or we could get a name for it. I prefer the former as it is on point!

Reply:
I have mixed the wording from both suggestions (check latest commit). Is that okay with you?